JDI.Light/JDI.Light/Elements/Common/Range.cs

```diff
@@ -11,13 +11,13 @@ public class Range : UIElement, IRange
         protected Range(By byLocator) : base(byLocator)
         {
         }
-
-        public void SetRange(string value)
+        
+        public void SetVolume(string value)
```

Comment:
@RomanIovlev , I think we should use Value here instead of Volume. 
Basically we are taking the data from `value` attribute:
![image](https://user-images.githubusercontent.com/6689884/56303674-1e453300-6145-11e9-9f8c-6cc38e7652dc.png)